src/components/HomeUpload/HomeUpload.js

```diff
@@ -31,22 +56,49 @@ export default function HomeUpload() {
       });
     };
 
+    const upload = async (formData, directory) => {
+      const uuid = shortid.generate();
+      const uploadUrl = `${apiUrl}/skynet/skyfile/${uuid}${
```

Comment:
we should drop uuid here

Reply:
don't forget to drop `shortid` dependency while you're at it

Reply:
good good